### DISCUSSION TITLE
error at any build of cloud functions

### DISCUSSION THREAD

**Discussion by guillo270120** [AntlerVC/firetable#1081] [Support / Q&A] [answered]
i have this error code. 

stderr: error Command failed with exit code 2.

i have a blaze plan, the custom functions was working, so time later the build always throw this error. I really need urgent support i have at production this application and i have to implement new functions to the backend.

**Comment by bofeiw** [comment, marked as answer]
Do you see anything wrong if you go to GCP cloud run console logging panel of rowy-backend? Usually the underlying can be found here.

**Reply by guillo270120** [reply]
![image](https://user-images.githubusercontent.com/104284249/212605907-4c5b14a3-3103-4238-b802-988a71cca04f.png)
![image](https://user-images.githubusercontent.com/104284249/212605960-a993ba20-7039-4c0e-9b14-18df754cd886.png)
this is from cloud run and from cloud functions

**Reply by guillo270120** [reply]
![image](https://user-images.githubusercontent.com/104284249/212607411-7999fa5e-469a-4547-b6fa-b020f723086e.png)
and this is my backend rowi roles

**Reply by guillo270120** [reply]
DERIVATIVE field cannot be copied using keyboard shortcut
2023-01-15 23:31:23:23.975 -06
0
23:31:24:24  Build started

1
23:31:24:24  Duplicating functions template to build/functionBuilder/builds/1673847084160

2
23:31:24:24  

3
23:31:24:24  Executing: cd build/functionBuilder/builds/1673847084160;pwd;ls -al

4
23:31:24:24  stdout: {"error":null,"stdout":"/home/node/app/build/functionBuilder/builds/1673847084160
total 8
drwxrwxrwx    2 root     root           160 Jan 16 05:31 .
drwxrwxr-x    2 root     root            60 Jan 16 05:31 ..
-rw-r--r--    1 root     root            43 Sep 12 14:04 firebase.json
drwxr-xr-x    2 root     root           360 Dec 31 07:39 lib
-rw-r--r--    1 root     root           578 Dec 31 07:36 package.json
drwxr-xr-x    2 root     root           360 Dec 31 07:39 src
-rw-r--r--    1 root     root           409 Nov 30  2021 tsconfig.json
-rw-r--r--    1 root     root          5131 Nov 30  2021 tslint.json
","stderr":""}

5
23:31:24:24  1

6
23:31:24:24  Executing: cd build/functionBuilder/builds/1673847084160;less package.json

7
23:31:24:24  stdout: {"error":null,"stdout":"{
  "name": "functions",
  "version": "0.0.1",
  "scripts": {
    "deploy": "echo 'n
' | firebase deploy --interactive"
  },
  "engines": {
    "node": "16"
  },
  "main": "src/index.js",
  "dependencies": {
    "@google-cloud/logging": "^10.3.1",
    "@google-cloud/secret-manager": "^4.1.4",
    "@google-cloud/storage": "6.7.0",
    "axios": "^1.2.0",
    "file-type": "^16.5.3",
    "firebase-admin": "^11.3.0",
    "firebase-functions": "^4.1.0",
    "lodash": "^4.17.21",
    "node-fetch": "^2.6.7",
    "uuid": "^8.3.2"
  },
  "devDependencies": {},
  "private": true
}
","stderr":""}

8
23:31:24:24  2

9
23:31:24:24  Executing: cd build/functionBuilder/builds/1673847084160/src;pwd;ls -al

10
23:31:24:24  stdout: {"error":null,"stdout":"/home/node/app/build/functionBuilder/builds/1673847084160/src
total 16
drwxr-xr-x    2 root     root           360 Dec 31 07:39 .
drwxrwxrwx    2 root     root           160 Jan 16 05:31 ..
drwxr-xr-x    2 root     root            80 Dec 31 07:39 derivatives
drwxr-xr-x    2 root     root            80 Dec 31 07:39 extensions
-rw-r--r--    1 root     root           704 Dec 31 07:39 firebaseConfig.js
-rw-r--r--    1 root     root           495 Dec 31 07:39 firebaseConfig.js.map
-rw-r--r--    1 root     root           156 Dec 31 07:39 functionConfig.js
-rw-r--r--    1 root     root           191 Dec 31 07:39 functionConfig.js.map
-rw-r--r--    1 root     root          4061 Dec 31 07:39 index.js
-rw-r--r--    1 root     root          2310 Dec 31 07:39 index.js.map
drwxr-xr-x    2 root     root            80 Dec 31 07:39 initialize
drwxr-xr-x    2 root     root            80 Dec 31 07:39 logging
drwxr-xr-x    2 root     root           160 Dec 31 07:39 propagates
-rw-r--r--    1 root     root          1027 Dec 31 07:39 rowy.js
-rw-r--r--    1 root     root           813 Dec 31 07:39 rowy.js.map
-rw-r--r--    1 root     root          1918 Dec 31 07:39 searchSync.js
-rw-r--r--    1 root     root          1636 Dec 31 07:39 searchSync.js.map
drwxr-xr-x    2 root     root           240 Dec 31 07:39 utils
","stderr":""}

11
23:31:24:24  3

12
23:31:24:24  Executing: cd build/functionBuilder/builds/1673847084160;yarn

13
23:31:40:40  stdout: {"error":null,"stdout":"info No lockfile found.
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
[4/5] Linking dependencies...
[5/5] Building fresh packages...
success Saved lockfile.
","stderr":"warning "firebase-admin > @firebase/database-compat > @firebase/database > @firebase/auth-interop-types@0.1.7" has unmet peer dependency "@firebase/app-types@0.x".
"}

14
23:31:40:40  Base dependencies installed successfully

15
23:31:40:40  Executing: cd build/functionBuilder/builds/1673847084160/node_modules/@google-cloud/logging/node_modules/@grpc/proto-loader;yarn

16
23:31:42:42  stdout: {"error":null,"stdout":"[1/5] Validating package.json...
[2/5] Resolving packages...
success Already up-to-date.
","stderr":"/bin/sh: cd: line 0: can't cd to build/functionBuilder/builds/1673847084160/node_modules/@google-cloud/logging/node_modules/@grpc/proto-loader: No such file or directory
"}

17
23:31:42:42  4

18
23:31:42:42  Executing: cd build/functionBuilder/builds/1673847084160/node_modules/@google-cloud/logging/node_modules/@grpc/proto-loader/build/src;pwd;ls -al

19
23:31:42:42  stdout: {"error":null,"stdout":"/home/node/app
total 336
drwxr-xr-x    2 root     root            60 Jan 16 05:31 .
drwxr-xr-x    2 node     node            60 Jan 16 05:31 ..
-rw-r--r--    1 root     root          6148 Nov 25 09:36 .DS_Store
-rw-r--r--    1 root     root            37 Dec 31 07:36 .dockerignore
-rw-r--r--    1 root     root            20 Nov 30  2021 .prettierignore
-rw-r--r--    1 root     root           644 Jul 18 09:46 Dockerfile
-rw-r--r--    1 root     root         11357 Nov 30  2021 LICENSE
-rw-r--r--    1 root     root          2012 Jul 18 09:46 README.md
drwxr-xr-x    2 root     root            60 Jan 16 05:31 build
-rwxr-xr-x    1 root     root           768 Dec 29 17:37 deploy.sh
-rw-r--r--    1 root     root          1535 Jul 18 09:46 icon.png
-rw-r--r--    1 root     root          3569 Jul 18 09:46 logo-sticker.svg
drwxr-xr-x    2 root     root             0 Dec 31 07:38 node_modules
-rw-r--r--    1 root     root           173 Jul 18 09:46 nodemon.json
-rw-r--r--    1 root     root          1538 Dec 31 07:36 package.json
drwxr-xr-x    2 root     root             0 Dec 31 07:38 src
-rwxr-xr-x    1 root     root           166 Jul 18 09:46 submit.sh
-rw-r--r--    1 root     root          6915 Nov 30  2021 tsconfig.json
-rw-r--r--    1 root     root        305036 Dec 31 07:36 yarn.lock
","stderr":"/bin/sh: cd: line 0: can't cd to build/functionBuilder/builds/1673847084160/node_modules/@google-cloud/logging/node_modules/@grpc/proto-loader/build/src: No such file or directory
"}

20
23:31:42:42  5

21
23:31:42:42  Generating schema...

22
23:31:42:42  derivativeColumns: [{"type":"DERIVATIVE","fieldName":"payload","index":18,"key":"payload","config":{"derivativeFn":"const derivative:Derivative = async ({row,ref,db,storage,auth,logging})=>{
  // WRITE YOUR CODE ONLY BELOW THIS LINE. DO NOT WRITE CODE/COMMENTS OUTSIDE THE FUNCTION BODY
  
  // Example:
   const sum = row.totalGravada + row.totalIgv;
   return sum;
  // WRITE YOUR CODE ONLY ABOVE THIS LINE. DO NOT WRITE CODE/COMMENTS OUTSIDE THE FUNCTION BODY
}","renderFieldType":"NUMBER","listenerFields":["factura"]},"name":"payload"}]

23
23:31:42:42  defaultValueColumns: []

24
23:31:42:42  documentSelectColumns: []

25
23:31:42:42  fieldTypes: {"totalGravada":"NUMBER","sunatTransaction":"NUMBER","clienteNumeroDeDocumento":"PHONE_NUMBER","porcentajeDeIgv":"NUMBER","total":"NUMBER","fechaDeEmision":"DATE_TIME","totalIgv":"NUMBER","clienteEmail":"EMAIL","uuid":"SIMPLE_TEXT","serie":"SIMPLE_TEXT","numero":"NUMBER","factura":"CHECK_BOX","moneda":"NUMBER","items":"SUB_TABLE","clienteDireccion":"SIMPLE_TEXT","tipoDeComprobante":"NUMBER","clienteTipoDeDocumento":"NUMBER","clienteDenominacion":"SIMPLE_TEXT","payload":"NUMBER"}

26
23:31:42:42  extensions: []

27
23:31:42:42  runtimeOptions: {}

28
23:31:42:42  tableSchema: {"columns":{"payload":{"type":"DERIVATIVE","fieldName":"payload","index":18,"key":"payload","config":{"derivativeFn":"const derivative:Derivative = async ({row,ref,db,storage,auth,logging})=>{
  // WRITE YOUR CODE ONLY BELOW THIS LINE. DO NOT WRITE CODE/COMMENTS OUTSIDE THE FUNCTION BODY
  
  // Example:
   const sum = row.totalGravada + row.totalIgv;
   return sum;
  // WRITE YOUR CODE ONLY ABOVE THIS LINE. DO NOT WRITE CODE/COMMENTS OUTSIDE THE FUNCTION BODY
}","renderFieldType":"NUMBER","listenerFields":["factura"]},"name":"payload"},"clienteDenominacion":{"index":0,"key":"clienteDenominacion","name":"Cliente Denominacion","type":"SIMPLE_TEXT","fieldName":"clienteDenominacion","config":{}},"clienteTipoDeDocumento":{"fieldName":"clienteTipoDeDocumento","name":"Cliente Tipo De Documento","config":{},"key":"clienteTipoDeDocumento","index":4,"type":"NUMBER"},"tipoDeComprobante":{"index":11,"fieldName":"tipoDeComprobante","config":{},"type":"NUMBER","name":"Tipo De Comprobante","key":"tipoDeComprobante"},"clienteDireccion":{"key":"clienteDireccion","fieldName":"clienteDireccion","index":1,"name":"Cliente Direccion","type":"SIMPLE_TEXT","config":{}},"items":{"type":"SUB_TABLE","fieldName":"items","index":16,"key":"items","config":{"parentLabel":["uuid"]},"name":"items"},"moneda":{"key":"moneda","type":"NUMBER","fieldName":"moneda","name":"Moneda","index":6,"config":{}},"factura":{"index":17,"type":"CHECK_BOX","config":{},"key":"factura","fieldName":"factura","name":"factura"},"numero":{"index":7,"name":"Numero","key":"numero","type":"NUMBER","fieldName":"numero","config":{}},"serie":{"fieldName":"serie","config":{},"name":"Serie","key":"serie","index":9,"type":"SIMPLE_TEXT"},"uuid":{"fieldName":"uuid","key":"uuid","index":15,"name":"Uuid","type":"SIMPLE_TEXT","config":{}},"clienteEmail":{"name":"Cliente Email","type":"EMAIL","index":2,"key":"clienteEmail","fieldName":"clienteEmail","config":{}},"totalIgv":{"index":14,"config":{},"key":"totalIgv","name":"Total Igv","fieldName":"totalIgv","type":"NUMBER"},"fechaDeEmision":{"name":"Fecha De Emision","type":"DATE_TIME","key":"fechaDeEmision","index":5,"fieldName":"fechaDeEmision","config":{}},"total":{"key":"total","fieldName":"total","index":12,"name":"Total","type":"NUMBER","config":{}},"porcentajeDeIgv":{"fieldName":"porcentajeDeIgv","config":{},"key":"porcentajeDeIgv","index":8,"type":"NUMBER","name":"Porcentaje De Igv"},"clienteNumeroDeDocumento":{"config":{},"key":"clienteNumeroDeDocumento","name":"Cliente Numero De Documento","fieldName":"clienteNumeroDeDocumento","index":3,"type":"PHONE_NUMBER"},"sunatTransaction":{"name":"Sunat Transaction","type":"NUMBER","config":{},"fieldName":"sunatTransaction","index":10,"key":"sunatTransaction"},"totalGravada":{"config":{},"index":13,"fieldName":"totalGravada","type":"NUMBER","name":"Total Gravada","key":"totalGravada"}},"functionConfigPath":"_rowy_/settings/functions/pagos"}

29
23:31:42:42  Generating config file...

30
23:31:43:43  Retrieving config file...

31
23:31:43:43  Validating config file...

32
23:31:43:43  Executing: cd build/functionBuilder/builds/1673847084160/src;tsc functionConfig.ts

33
23:31:58:58  stdout: {"error":null,"stdout":"","stderr":""}

34
23:31:58:58  

35
23:31:58:58  Config file: "import fetch from "node-fetch";
import rowy from "./rowy";

export const fieldTypes = {
  "totalGravada": "NUMBER",
  "sunatTransaction": "NUMBER",
  "clienteNumeroDeDocumento": "PHONE_NUMBER",
  "porcentajeDeIgv": "NUMBER",
  "total": "NUMBER",
  "fechaDeEmision": "DATE_TIME",
  "totalIgv": "NUMBER",
  "clienteEmail": "EMAIL",
  "uuid": "SIMPLE_TEXT",
  "serie": "SIMPLE_TEXT",
  "numero": "NUMBER",
  "factura": "CHECK_BOX",
  "moneda": "NUMBER",
  "items": "SUB_TABLE",
  "clienteDireccion": "SIMPLE_TEXT",
  "tipoDeComprobante": "NUMBER",
  "clienteTipoDeDocumento": "NUMBER",
  "clienteDenominacion": "SIMPLE_TEXT",
  "payload": "NUMBER"
}
export const triggerPath = "pagos/{docId}"
export const functionName = "pagos"
export const derivativesConfig = [{
  fieldName: 'payload',
  requiredPackages: [],
  evaluate: async ({
    row,
    ref,
    db,
    auth,
    storage,
    utilFns,
    logging
  }) => {
    // WRITE YOUR CODE ONLY BELOW THIS LINE. DO NOT WRITE CODE/COMMENTS OUTSIDE THE FUNCTION BODY

    // Example:
    const sum = row.totalGravada + row.totalIgv;
    return sum;
    // WRITE YOUR CODE ONLY ABOVE THIS LINE. DO NOT WRITE CODE/COMMENTS OUTSIDE THE FUNCTION BODY
  },
  listenerFields: ["factura"]
}, ]
export const defaultValueConfig = []
export const documentSelectConfig = []
export const extensionsConfig = []
export const runtimeOptions = {
  "serviceAccount": "rowy-functions@app-icap.iam.gserviceaccount.com"
}
export const tableSchema = {
  "columns": {
    "payload": {
      "type": "DERIVATIVE",
      "fieldName": "payload",
      "index": 18,
      "key": "payload",
      "config": {
        "derivativeFn": "const derivative:Derivative = async ({row,ref,db,storage,auth,logging})=>{
  // WRITE YOUR CODE ONLY BELOW THIS LINE. DO NOT WRITE CODE/COMMENTS OUTSIDE THE FUNCTION BODY
  
  // Example:
   const sum = row.totalGravada + row.totalIgv;
   return sum;
  // WRITE YOUR CODE ONLY ABOVE THIS LINE. DO NOT WRITE CODE/COMMENTS OUTSIDE THE FUNCTION BODY
}",
        "renderFieldType": "NUMBER",
        "listenerFields": ["factura"]
      },
      "name": "payload"
    },
    "clienteDenominacion": {
      "index": 0,
      "key": "clienteDenominacion",
      "name": "Cliente Denominacion",
      "type": "SIMPLE_TEXT",
      "fieldName": "clienteDenominacion",
      "config": {}
    },
    "clienteTipoDeDocumento": {
      "fieldName": "clienteTipoDeDocumento",
      "name": "Cliente Tipo De Documento",
      "config": {},
      "key": "clienteTipoDeDocumento",
      "index": 4,
      "type": "NUMBER"
    },
    "tipoDeComprobante": {
      "index": 11,
      "fieldName": "tipoDeComprobante",
      "config": {},
      "type": "NUMBER",
      "name": "Tipo De Comprobante",
      "key": "tipoDeComprobante"
    },
    "clienteDireccion": {
      "key": "clienteDireccion",
      "fieldName": "clienteDireccion",
      "index": 1,
      "name": "Cliente Direccion",
      "type": "SIMPLE_TEXT",
      "config": {}
    },
    "items": {
      "type": "SUB_TABLE",
      "fieldName": "items",
      "index": 16,
      "key": "items",
      "config": {
        "parentLabel": ["uuid"]
      },
      "name": "items"
    },
    "moneda": {
      "key": "moneda",
      "type": "NUMBER",
      "fieldName": "moneda",
      "name": "Moneda",
      "index": 6,
      "config": {}
    },
    "factura": {
      "index": 17,
      "type": "CHECK_BOX",
      "config": {},
      "key": "factura",
      "fieldName": "factura",
      "name": "factura"
    },
    "numero": {
      "index": 7,
      "name": "Numero",
      "key": "numero",
      "type": "NUMBER",
      "fieldName": "numero",
      "config": {}
    },
    "serie": {
      "fieldName": "serie",
      "config": {},
      "name": "Serie",
      "key": "serie",
      "index": 9,
      "type": "SIMPLE_TEXT"
    },
    "uuid": {
      "fieldName": "uuid",
      "key": "uuid",
      "index": 15,
      "name": "Uuid",
      "type": "SIMPLE_TEXT",
      "config": {}
    },
    "clienteEmail": {
      "name": "Cliente Email",
      "type": "EMAIL",
      "index": 2,
      "key": "clienteEmail",
      "fieldName": "clienteEmail",
      "config": {}
    },
    "totalIgv": {
      "index": 14,
      "config": {},
      "key": "totalIgv",
      "name": "Total Igv",
      "fieldName": "totalIgv",
      "type": "NUMBER"
    },
    "fechaDeEmision": {
      "name": "Fecha De Emision",
      "type": "DATE_TIME",
      "key": "fechaDeEmision",
      "index": 5,
      "fieldName": "fechaDeEmision",
      "config": {}
    },
    "total": {
      "key": "total",
      "fieldName": "total",
      "index": 12,
      "name": "Total",
      "type": "NUMBER",
      "config": {}
    },
    "porcentajeDeIgv": {
      "fieldName": "porcentajeDeIgv",
      "config": {},
      "key": "porcentajeDeIgv",
      "index": 8,
      "type": "NUMBER",
      "name": "Porcentaje De Igv"
    },
    "clienteNumeroDeDocumento": {
      "config": {},
      "key": "clienteNumeroDeDocumento",
      "name": "Cliente Numero De Documento",
      "fieldName": "clienteNumeroDeDocumento",
      "index": 3,
      "type": "PHONE_NUMBER"
    },
    "sunatTransaction": {
      "name": "Sunat Transaction",
      "type": "NUMBER",
      "config": {},
      "fieldName": "sunatTransaction",
      "index": 10,
      "key": "sunatTransaction"
    },
    "totalGravada": {
      "config": {},
      "index": 13,
      "fieldName": "totalGravada",
      "type": "NUMBER",
      "name": "Total Gravada",
      "key": "totalGravada"
    }
  },
  "functionConfigPath": "_rowy_/settings/functions/pagos"
}
export const region = "us-central1"
export const searchHost = null
export const searchIndices = []"

36
23:31:58:58  Installing dependencies: []

37
23:31:58:58  Installing extensions: []

38
23:31:59:59  Installing dependencies...

39
23:31:59:59  [1/5] Validating package.json...
[2/5] Resolving packages...
success Already up-to-date.

40
23:32:00:00  Deploying pagos to app-icap

41
23:32:00:00  Executing: cd build/functionBuilder/builds/1673847084160; yarn deploy --project app-icap --only functions

42
23:33:52:52  stdout: {"error":{"code":2,"killed":false,"signal":null,"cmd":"cd build/functionBuilder/builds/1673847084160; yarn deploy --project app-icap --only functions"},"stdout":"$ echo 'n
' | firebase deploy --interactive --project app-icap --only functions

=== Deploying to 'app-icap'...

i  deploying functions
i  functions: ensuring required API cloudfunctions.googleapis.com is enabled...
i  functions: ensuring required API cloudbuild.googleapis.com is enabled...
i  artifactregistry: ensuring required API artifactregistry.googleapis.com is enabled...
✔  artifactregistry: required API artifactregistry.googleapis.com is enabled
✔  functions: required API cloudfunctions.googleapis.com is enabled
✔  functions: required API cloudbuild.googleapis.com is enabled
i  functions: preparing codebase default for deployment
i  functions: preparing . directory for uploading...
i  functions: packaged /home/node/app/build/functionBuilder/builds/1673847084160 (107.13 KB) for uploading
✔  functions: . folder uploaded successfully

The following functions are found in your project but do not exist in your local source code:
	R-detallePago(us-central1)
	addFcmToken(us-central1)
	ffPrivateApiCall(us-central1)
	sendPushNotificationsTrigger(us-central1)
	sendUserPushNotificationsTrigger(us-central1)

If you are renaming a function or changing its region, it is recommended that you create the new function first before deleting the old one to prevent event loss. For more info, visit https://firebase.google.com/docs/functions/manage-functions#modify

? Would you like to proceed with deletion? Selecting no will continue the rest 
of the deployments. (y/N) u001b[1Au001b[26Du001b[105Cu001b[1Bu001b[2Ku001b[1Au001b[2Ku001b[G? Would you like to proceed with deletion? Selecting no will continue the rest 
of the deployments. (y/N) nu001b[1Au001b[27Du001b[106Cu001b[1Bu001b[2Ku001b[1Au001b[2Ku001b[G? Would you like to proceed with deletion? Selecting no will continue the rest 
of the deployments. Nou001b[1Au001b[22Du001b[101C
r
u001b[1Bi  functions: updating Node.js 16 function R-pagos(us-central1)...
Failed to configure trigger event-type:providers/cloud.firestore/eventTypes/document.write resource:projects/app-icap/databases/(default)/documents/pagos/{docId} service:firestore.googleapis.com

Functions deploy had errors with the following functions:
	R-pagos(us-central1)
i  functions: cleaning up build files...

Error: There was an error deploying functions
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
","stderr":"error Command failed with exit code 2.
"}

43
23:33:52:52  $ echo 'n
' | firebase deploy --interactive --project app-icap --only functions

=== Deploying to 'app-icap'...

i  deploying functions
i  functions: ensuring required API cloudfunctions.googleapis.com is enabled...
i  functions: ensuring required API cloudbuild.googleapis.com is enabled...
i  artifactregistry: ensuring required API artifactregistry.googleapis.com is enabled...
✔  artifactregistry: required API artifactregistry.googleapis.com is enabled
✔  functions: required API cloudfunctions.googleapis.com is enabled
✔  functions: required API cloudbuild.googleapis.com is enabled
i  functions: preparing codebase default for deployment
i  functions: preparing . directory for uploading...
i  functions: packaged /home/node/app/build/functionBuilder/builds/1673847084160 (107.13 KB) for uploading
✔  functions: . folder uploaded successfully

The following functions are found in your project but do not exist in your local source code:
	R-detallePago(us-central1)
	addFcmToken(us-central1)
	ffPrivateApiCall(us-central1)
	sendPushNotificationsTrigger(us-central1)
	sendUserPushNotificationsTrigger(us-central1)

If you are renaming a function or changing its region, it is recommended that you create the new function first before deleting the old one to prevent event loss. For more info, visit https://firebase.google.com/docs/functions/manage-functions#modify

? Would you like to proceed with deletion? Selecting no will continue the rest 
of the deployments. (y/N) ? Would you like to proceed with deletion? Selecting no will continue the rest 
of the deployments. (y/N) n? Would you like to proceed with deletion? Selecting no will continue the rest 
of the deployments. No

i  functions: updating Node.js 16 function R-pagos(us-central1)...
Failed to configure trigger event-type:providers/cloud.firestore/eventTypes/document.write resource:projects/app-icap/databases/(default)/documents/pagos/{docId} service:firestore.googleapis.com

Functions deploy had errors with the following functions:
	R-pagos(us-central1)
i  functions: cleaning up build files...

Error: There was an error deploying functions
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

44
23:33:52:52  

command: cd build/functionBuilder/builds/1673847084160; yarn deploy --project app-icap --only functions

stderr: error Command failed with exit code 2.


errorStackTrace: Error: Command failed: cd build/functionBuilder/builds/1673847084160; yarn deploy --project app-icap --only functions
error Command failed with exit code 2.

    at ChildProcess.exithandler (node:child_process:402:12)
    at ChildProcess.emit (node:events:513:28)
    at maybeClose (node:internal/child_process:1100:16)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:304:5)

2023-01-15 22:54:07:07.888 -06
2023-01-15 21:51:34:34.041 -06
2023-01-15 20:53:00:00.646 -06
2023-01-15 20:49:56:56.958 -06
2023-01-15 20:35:37:37.194 -06
2023-01-15 19:59:14:14.789 -06
2023-01-15 19:51:44:44.975 -06
2023-01-15 19:45:48:48.846 -06
2023-01-15 15:42:16:16.608 -06
2023-01-15 15:37:32:32.701 -06
2023-01-15 15:25:19:19.670 -06
2023-01-15 15:19:14:14.245 -06
2023-01-15 14:18:00:00.443 -06
2023-01-15 14:14:28:28.787 -06

**Reply by BeeBombshell** [reply]
Hey @guillo270120, I think it'd be best if you could book a [short 1:1 meet](https://rowy.io/meet) so that we can go over the issue and get it fixed for you.

**Reply by mravani2** [reply]
I'm facing the same issue. @BeeBombshell where you able to help him fix the problem?

**Reply by BeeBombshell** [reply]
Hey @mravani2, he might have had a meeting with @shamsmosowi. Tagging him here so that he can help you out as well.
Since the script was running seamlessly before, I am assuming the issue is related to the imported npm packages which might have been updated and now seem to be incompatible. Could you check if that is the case?

**Reply by mravani2** [reply]
@BeeBombshell how can I check this?
